Lib/asyncio/streams.py

```diff
@@ -303,6 +315,13 @@ def can_write_eof(self):
     def close(self):
         return self._transport.close()
 
+    def is_closing(self):
+        return self._transport.is_closing()
+
+    async def wait_closed(self):
+        if self._transport is not None:
+            await self._protocol._closed
```

Comment:
When is the transport None?  Can you add a comment?

Reply:
Actually I have no idea, just copied the line from `drain`.
Looking on source code there is no line for setting transport to `None`.
Let's drop the check from both methods.